lua/modules/linear_bolzmann_solvers/do_transient/lbts_lua_advance_time_data.cc

```diff
@@ -33,12 +33,12 @@ LBTSAdvanceTimeData(lua_State* L)
   const int solver_handle = lua_tointeger(L, 1);
 
   auto& solver =
-    opensn::GetStackItem<lbs::DiscOrdTransientSolver>(opensn::object_stack, solver_handle, fname);
+    opensn::GetStackItem<DiscOrdTransientSolver>(opensn::object_stack, solver_handle, fname);
```

Comment:
Ouch, never knew the unused transient solver is `DiscOrdTransientSolver`. We should change this to `DiscreteOrdinatesTransientSolver` when we get around to it

Reply:
I added this as a task into #57 which is dedicated to transient capability